libs/csm/rest/csm_rest_quota.py

```diff
@@ -116,32 +125,32 @@ def verify_get_set_user_quota(self, uid, payload, verify_response=False,
                     if key in ("enabled","max_size","max_objects","check_on_raw"):
                         continue
                     if key == "max_size_kb":
-                        payload.update({"max_size_kb":max_size/1000})
+                        payload.update({"max_size_kb":get_response["max_size"]/1000})
                     self.log.info("Actual response for %s: %s", key, payload[key])
                     if value != payload[key]:
                         self.log.error("Actual and expected response for %s didnt match", key)
                         result = False
         else:
             self.log.error("Status code check failed.")
             result = False
-        return result, resp
+        return result, get_response
 
-    def verify_max_size(self, max_size):
+    def verify_max_size(self, max_size, akey, skey):
         """
         Perform put operation with 1 object and then perform put
         of random size and check if that fails
         """
         self.log.info("Perform Put operation for 1 object of max size")
         resp = s3_misc.create_put_objects(self.obj_name, self.bucket,
-                                          self.akey, self.skey, object_size=max_size)
-        assert_utils.assert_true(resp, "Put object Failed")
+                                          akey, skey, object_size=max_size)
+        assert resp, "Put object Failed"
         self.log.info("Perform Put operation of Random size and 1 object")
         random_size = self.cryptogen.randrange(1, max_size)
         resp = s3_misc.create_put_objects(self.obj_name, self.bucket,
-                                          self.akey, self.skey, object_size=random_size)
-        assert_utils.assert_false(resp, "Put object did not fail")
+                                          akey, skey, object_size=random_size)
+        assert resp, "Put object did not fail"
```

Comment:
It should be assert not resp....

Also Can u please make the error message more clear
"User is able to perform put object after exceeding ???"

Reply:
done